templates/vue/.gitignore

```diff
@@ -4,6 +4,8 @@ npm-debug.log
 yarn-error.log
 config.js
 cypress.json
+cypress/screenshots/
+cypress/videos/
```

Comment:
These should be removed since cypress is no longer here, right?

Reply:
Cypress still generates these folders despite running inside the container, so I think it should be there still.